packages/flutter_test/lib/src/binding.dart

```diff
@@ -700,13 +751,43 @@ abstract class TestWidgetsFlutterBinding extends BindingBase
     // The LiveTestWidgetsFlutterBinding overrides this to report the exception to the console.
   }
 
+  Future<dynamic> _handleMessage(dynamic mockMessage) async {
+    final Map<dynamic, dynamic> message = mockMessage as Map<dynamic, dynamic>;
```

Comment:
Here and elsewhere: `Object?` instead of dynamic

Reply:
Done!